### ISSUE TITLE
Photos are loaded right to left

### ISSUE THREAD

**Issue by flpommeton** [Dynalon/mdwiki#223]
Is there a way to force photos to load from left to right? Right to left makes chronological sets difficult.

**Comment by Dynalon** [comment]
You can chose to sort from left-to-right or from right-to-left by placing the images at the top of the paragraph (left-to-right) or at the bottom (right-to-left). See the Layout example: http://dynalon.github.io/mdwiki/#!layout.md#Floating_to_the_left

**Comment by flpommeton** [comment]
Thanks for getting back to me. I was of the impression that this technique would control the float, but I did not realize that it also controlled the display order within the Lightbox. Usually if I am displaying a few photos the Lightbox displays the images from somewhere in the middle of the list of photos.

**Comment by flpommeton** [comment]
This is still not working the way that I think that it should work (and maybe not the way you think). If I arrange text and photos like this:

Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
![p1](http://www.~/p1.jpg "")
![p2](http://www.~/p2.jpg "")
![p3](http://www.~/p3.jpg "")
![p4](http://www.~/p4.jpg "")
![p5](http://www.~/p5.jpg "")
![p6](http://www.~/p6.jpg "")

The photos display in reverse order as follows:
Row 1:
![p5](http://www.~/p5.jpg "")
![p4](http://www.~/p4.jpg "")
![p3](http://www.~/p3.jpg "")
![p2](http://www.~/p2.jpg "")
![p1](http://www.~/p1.jpg "")
Row 2:
![p6](http://www.~/p6.jpg "")

This makes it difficult to determine how to arrange the lists of photos to get them to display back in a chronological order.

If I am missing something more, please let me know.

**Comment by Dynalon** [comment]
Hm then this seems like a bug. As a workaround you could reverse the order of the images (oldest first) and it should diplay it in newest-first order?

**Comment by flpommeton** [comment]
Yes, but I may have mislead with the term "chronological order". I probably should have said something like "presentation order". It is really the order that I would arrange the photos in LR for presentation. I could rearrange/fiddle them in the markdown file but I sometimes have 20 or more photos in a list and I would have to be calculating the reorder every 5 slides or so. Not sure how this would work in the long run. I do not mean to be a complainer, but I may be too lazy to do this everytime. If you have a quick look at July 1 in www.appletons.ca you will see a list of 14 photos of a farm. They are reordering every 5 photos.
